app.js

```diff
@@ -1,9 +1,12 @@
+require('dotenv').config()
```

Comment:
I like how you made this DRY by putting it in `app.js` rather than the specific route file.